t/t-pre-push.sh

```diff
@@ -363,7 +363,7 @@ begin_test "pre-push allow missing object (found on server)"
     tee push.log
 
   # make sure there were no errors reported
-  [ -z "$(grep -i 'Error' push.log)" ]
+  [ 0 -eq "$(grep -c -i 'Error' push.log)" ]
```

Comment:
Nit: I think this would succeed if the `push.log` is not created due to a previous error. We likely would abort with a previous error and therefore that should be no issue.